aicssegmentation/structure_wrapper_config/all_functions.json

```diff
@@ -1,8 +1,9 @@
 {
-    "Intensity Normalization": {
-        "module": "aicssegmentation.core.pre_processing_utils",
-        "function": "intensity_normalization",
-        "parameter": {
+    "intensity_normalization": {
+        "name": "Intensity Normalization",
+        "python::module": "aicssegmentation.core.pre_processing_utils",
```

Comment:
is there any special meaning for  `: :` ? just curious.

Reply:
No special meaning. I added this to make it obvious that these properties are referring to python modules / python functions. The reason I did this is because it was getting a bit confusing when we have several places referring to a `function` both in the config and in the code. Are we talking about a "segmenter function" or an actual python function? This just makes it obvious and removes any confusion. It's just a readability thing.